Content.Shared/Movement/Components/SpeedModifierContactsComponent.cs

```diff
@@ -13,10 +13,14 @@ public sealed partial class SpeedModifierContactsComponent : Component
     [AutoNetworkedField]
     public float WalkSpeedModifier = 1.0f;
 
-    [AutoNetworkedField]
     [DataField("sprintSpeedModifier"), ViewVariables(VVAccess.ReadWrite)]
+    [AutoNetworkedField]
     public float SprintSpeedModifier = 1.0f;
 
+    [DataField("affectAirborne"), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
    [DataField, ViewVariables(VVAccess.ReadWrite)]
```
is a deprecated variable serialization format. It is unnecessary, because now the variable name with a lowercase letter is automatically used